cloudzero/README.md

```diff
@@ -2,45 +2,44 @@
 
 ## Setup
 
 ### Installation
 
-Log in to your [CloudZero account](https://app.cloudzero.com) and navigate to the Connections settings. Create a new connection to Datadog. Once a connection is created, you will click "Authorize Datadog Account" to authorize CloudZero to pull data from your Datadog account into the CloudZero platform.
+Log in to your [CloudZero account][4] and navigate to the Connections settings. Create a new connection to Datadog. 
+
+Once a connection is created, click **Authorize Datadog Account** to authorize CloudZero to pull data from your Datadog account into the CloudZero platform.
 
-Alternatively, you can initiate the CloudZero/Datadog integration by clicking Install
-Integration on the Datadog integration tile. Next, click **Connect Accounts** under 
-the Configure tab to begin the OAuth workflow and create a connection in the CloudZero
-platform.
+Alternatively, you can initiate the CloudZero/Datadog integration by clicking **Install Integration** on the [Datadog integration tile][1]. Next, click **Connect Accounts** under the Configure tab to begin the OAuth workflow and create a connection in the CloudZero platform.
 
 ### Configuration
 
-You will be able to adjust settings from the connection details page for your
-Datadog connection in CloudZero.
+You can adjust settings from the connection details page for your Datadog connection in CloudZero.
 
 ### Validation
 
-1. From the list of [connections](http://app.cloudzero.com/organization/connections), you will be able to see the status of your connection to Datadog. 
-2. Click on the name of your Datadog connection to view more details about the
-   amount and timing of data that has been pulled from Datadog.
-3. Once data ingestion has successfully run, you will see Datadog costs included in the
-   [Cost Explorer](http://app.cloudzero.com/explorer). 
+1. From the list of [connections][5], you can see the status of your connection to Datadog. 
+2. Click on the name of your Datadog connection to view more details about the amount and timing of data that has been pulled from Datadog.
+3. Once data ingestion has successfully run, you will see Datadog costs included in the [Cost Explorer][6].
 
 ### Uninstallation
 - Once this integration has been uninstalled, any previous authorizations are revoked.
-- Additionally, ensure that all API keys associated with this integration have been disabled by searching for the integration name on the [API Keys page](https://app.datadoghq.com/organization-settings/api-keys).
+- Additionally, ensure that all API keys associated with this integration have been disabled by searching for the integration name on the [API Keys page][7].
 
 ## Troubleshooting
 
 Need help? Contact [CloudZero support][3].
 
-[1]: **LINK_TO_INTEGRATION_SITE**
+[1]: https://app.datadoghq.com/integrations/cloudzero
 [2]: https://app.datadoghq.com/account/settings#agent
 [3]: mailto:support@cloudzero.com
-
+[4]: https://app.cloudzero.com
```

Comment:
These are out of order, but I don't believe we care about that and they do seem to match up. If there's any context in which our users view this as an .md, though, we should reorder.